geniza/corpus/admin.py

```diff
@@ -126,6 +128,7 @@ class DocumentAdmin(admin.ModelAdmin):
         'doctype', 'languages',
         'probable_languages',
         'status',
+        ('footnotes__content', admin.EmptyFieldListFilter),
```

Comment:
this works but is labeled kinda weirdly, is there a better way?

Reply:
Pretty sure we need it on the doc view; we should ask the team. Being able to see if a document is transcribed or not seems like core functionality.

Reply:
I agree they need a way to see that, just not sure if we need it _here_. If it doesn't slow things down too much then let's keep it, since you've already implemented. (But we need one on the footnote view as well, and you'll have to figure out how to fix the label)